utils/docker/Dockerfile.centos.7

```
@@ -98,31 +98,36 @@ ARG JENKINS_URL=""
 ARG QUICKBUILD=false
 ARG QUICKBUILD_DEPS=""
 ARG REPOS=""
-RUN if $QUICKBUILD; then                                                         \
-        echo -e "[repo.dc.hpdd.intel.com_repository_daos-stack-el-7-x86_64-stable-local]\n\
-name=created by dnf config-manager from https://repo.dc.hpdd.intel.com/repository/daos-stack-el-7-x86_64-stable-local\n\
-baseurl=https://repo.dc.hpdd.intel.com/repository/daos-stack-el-7-x86_64-stable-local\n\
+ARG REPO_URL=""
+ARG REPO_EL7=""
+RUN if $QUICKBUILD; then                                                      \
+        MY_REPO="${REPO_URL}${REPO_EL7}";                                     \
+        MY_NAME1="${MY_REPO#*//}";                                            \
+        MY_NAME="${MY_NAME1//\//_}";                                          \
+        echo -e "[${MY_NAME}]\n\
+name=created by dnf config-manager from ${MY_REPO}\n\
```

Let's make this a bit more accurate:
```suggestion
name=created from ${MY_REPO}\n\
```

---

Will change.